packages/app/hooks/use-drop-nft.ts

```diff
@@ -149,7 +149,8 @@ export const useDropNFT = () => {
         await pollTransaction(relayerResponse.relayed_transaction_id);
       } else {
         Alert.alert(
-          "Wallet seems to be disconnected. Try signing out and signing in."
+          "Wallet disconnected",
+          "Please logout and login again to complete the transaction"
```

Comment:
I think somehow we should handle and solve this later because it doesn't look like a good approach to say user logout and log in :/

Reply:
Also will you handle Twitter colored button like Alex mentioned? @intergalacticspacehighway 
https://showtime-rq88331.slack.com/archives/C02PXGK3V8D/p1655138669237659

Reply:
> I think somehow we should handle and solve this later because it doesn't look like a good UX to say logout and log in :/


yeah working on a hook that'll handle it


> Also will you handle Twitter colored button like Alex mentioned? @intergalacticspacehighway


Yes, added it here https://github.com/showtime-xyz/showtime-frontend/pull/1171/files#diff-be9253eac967b37574365765760354328be2b22357666e4aa7d6b55c3f46add2R105

Reply:
Oh sorry missed 👍🏽